frontend/components/auction/debt/DebtAuction.vue

```diff
@@ -0,0 +1,259 @@
+                <AuctionRestartPanel
+                    :wallet-address="walletAddress"
+                    :is-explanations-shown="isExplanationsShown"
+                    :is-restarting="auctionActionState === 'restarting'"
+                    :is-connecting="isConnecting"
+                    auction-type="surplus"
+                    @restart="$emit('restart', auctionId)"
+                    @connectWallet="$emit('connect')"
+                    @disconnectWallet="$emit('disconnect')"
+                />
+            </div>
+            <div class="relative mt-2">
+                <table class="w-full table-fixed border-collapse border">
+                    <tbody>
+                        <tr>
+                            <td>Auction State</td>
+                            <td>
+                                <span v-if="isActive">Ends in </span>
+                                <span v-else-if="requiresRestart">Requires restart</span>
+                                <span v-else-if="auction.state === 'ready-for-collection'">Ended </span>
+                                <span v-else>Collected </span>
+                                <time-till
+                                    v-if="auction.earliestEndDate && !requiresRestart"
+                                    :date="auction.earliestEndDate"
+                                />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Auction Fixed Bid</td>
+                            <td>
+                                <format-currency :value="auction.receiveAmountDAI" currency="DAI" />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Current Compensation</td>
+                            <td v-if="withBids">
+                                <format-currency :value="auction.bidAmountMKR" currency="MKR" />
+                            </td>
+                            <td v-else>
+                                <span class="opacity-50">No bids yet</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Auction Price</td>
+                            <td>
+                                <template v-if="withBids">
+                                    <format-currency :value="auction.unitPrice" :decimal-places="6" currency="DAI" />
+                                    per <format-currency currency="MKR" />
+                                </template>
+                                <span v-else class="opacity-50">No bids yet</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Price On Uniswap</td>
+                            <td>
+                                <template v-if="auction.marketUnitPrice">
+                                    <format-currency
```

Comment:
@LukSteib, In this component (and the mock) we have the Auction Price be displayed as:

"__ DAI per MKR" 

and the Price on Uniswap is displayed as:

"___ MKR per DAI". 

I would change these values to use the same measurement. Preferably "___ MKR per DAI", as DAI is always 1 USD, making it easier to understand and see the value.

Reply:
Error in Mock: For the debt auction we always go with the "DAI per MKR" connotation since this is the exchange that happens (user gives DAI and receives MKR)